### PR TITLE
Create visualization classes playground

### PR DESCRIPTION
**Pull request by pockerman**
Checkout this playground of what I have in mind and let me know what you think

### CODE REVIEW

File: src/main/java/visualizations/LineChart.java [hunk 1]
```diff
@@ -14,6 +16,23 @@
 
 public class LineChart {
 
+    public class LinePlotOptions
```
Comment (jethronap) [inline src/main/java/visualizations/LineChart.java:19]: ok this class is public. but what about the parameters. should we make them private and have some getters & setters? is it getting more verbose that way? perhaps use Lombok?

**Review by jethronap — COMMENTED** [review]
check out my comments, pls.

File: src/main/java/visualizations/LineChart.java (hunk 1 above)
Comment (pockerman) [reply src/main/java/visualizations/LineChart.java:19]: I would say leave it public...what is Lombok?

Comment (jethronap) [reply src/main/java/visualizations/LineChart.java:19]: I am not sure that we should leave it public bu ok for now. [Lombok](https://projectlombok.org) nice one! helps with over typing check it out.

Comment (pockerman) [reply src/main/java/visualizations/LineChart.java:19]: In this case we just want to take some variables from one place to another that's all. This class is not meant to do smart things just moving data from one place to another. So I think it's ok having it like this

Comment (jethronap) [reply src/main/java/visualizations/LineChart.java:19]: agreed. will refactor the other classes too on my branch.

**Comment by jethronap** [comment]
we don't merge. just closing this PR. right?

**Comment by pockerman** [comment]
should I close this PR???

**Pull request closed without merging**